guides/asynchronous-tasks/readme.md

```diff
@@ -364,21 +366,23 @@ require 'async'
 require 'thread/local' # thread-local gem.
 
 class TimeCache
-	extend Thread::Local
+	extend Thread::Local # defines `instance` class method that lazy-creates a separate instance per thread
 	
 	def initialize
 		@current_time = nil
 	end
 	
-	def current_time
+	def current_time_string
 		refresh!
 		
 		return @current_time
 	end
 	
-	private def refresh!
+	private
```

Comment:
`private` on the same line as `def` is really unusual so I figured it was best to avoid here.